json_serializable/test/kitchen_sink/kitchen_sink.wrapped.g.dart

```diff
@@ -67,6 +72,7 @@ abstract class _$KitchenSinkSerializerMixin {
   bool get writeNotNull;
   String get string;
   SimpleObject get simpleObject;
+  StrictKeysObject get strictKeysObject;
```

Comment:
nit: it'd be better to put this in the json_serial_example thingy

...this thing is SOOO big. Just update an existing class w/ the annotation to check behavior...

Reply:
The point is that it literally tests everything though right? Fwiw adding this here did surface an actual error... and it gives coverage for all the combinations of other options which I think is good.

What is the `json_serial_example` thingy?

Reply:
going to submit as is for now it should be trivial to make this change as a followup if desired